portfolio/src/main/webapp/script.js

```diff
@@ -15,7 +15,10 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
+function addRandomFact() {
+  const defaultBackgroundcolor = "rgb(40, 255, 225)";
```

Comment:
Can you modify this to read the starting background color, and only reset it if has been changed? Defining the background constant in both the js and css seems brittle, as there is the potential for them to be changed independently and come out of sync.

Reply:
Oh yeah I see that now. I made a default variable in css that the js calls when updating the background color to it's original color